WordPress/src/main/res/layout/login_epilogue_screen_new.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ProgressBar
+        android:id="@+id/sites_progress"
+        style="?android:attr/progressBarStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:visibility="visible" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycler_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:scrollbars="vertical"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <FrameLayout
+        android:id="@+id/orLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_extra_medium_large"
+        android:paddingBottom="@dimen/margin_medium"
+        android:paddingTop="@dimen/margin_medium"
+        app:layout_constraintBottom_toTopOf="@+id/bottom_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/recycler_view">
+
+        <TextView
+            style="@style/Widget.LoginFlow.TextView.DividerText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:paddingEnd="@dimen/margin_small"
+            android:paddingStart="@dimen/margin_small"
+            android:text="@string/login_or" />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:layout_gravity="center_vertical"
+            android:alpha="0.12"
+            android:background="?attr/colorOnSurface" />
+
+    </FrameLayout>
```

Comment:
Suggestion 💡: Since this `orLayout` is now used at three places:

`login_email_screen.xml`
`login_epilogue_screen_new.xml`
`login_epilogue_screen_new_expanded.xml`

how about it is extracted to a separate layout and included in all three places? This way, we wouldn't have to worry about the dividing line going through the "or" text as noticed by Chris in the future.

Reply:
👋 @ashiagr !

Such a nice suggestion, totally, I agree 💯 and will do that asap!

Reply:
👋 @ashiagr !

I have created this [WordPress-Login-Flow-Android#60](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/60) PR which extract the `login-or-layout` as discussed. Let me know your thoughts. Then, I'll incorporate this change within this PR.

PS: @osullivanchris , this [WordPress-Login-Flow-Android#60](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/60) PR above slightly changes the UI on the `login-email-screen` for just a `2dp` bottom diff, between the `Or Layout` and the `Google Sign-In` button. This change is barely noticeable but I chose to make it to make this `Or Layout` consistent across all screens afterwards. Are you okay with this change? PS: When this change gets incorporated in this PR I can ping you again so that you can double check the `login-email-screen` and see if you don't like something, maybe those extra `2dp` will create an issue of some kind, design wise.

Reply:
@ParaskP7 that sounds fine. Good to be consistent and stick to the 8dp grid. But I can take a look at a build/screenshot of both screens when its done just to be sure and give it a final 👍

Reply:
Yes, exactly, true. I'll then ping you when we get ready with the final APK and I'll have the updated screenshots.

PS: Bear in mind that this might take sometime since we are having some issues with the `Login` library but `Platform9` is working on fixing them as we speak.

Reply:
👋 @ashiagr !

I have updated the PR with the extracted `login-or-layout` that is not being reused, could you please do another review on that: fbc7089c325b5aefee0c579dd102d1c1665655f0 🙏 

👋 @osullivanchris !

I have update the PR description screenshots to the how the `Or Layout` is looking now. Can you also please run through the app and verify that everything works as expected, including looking at the `login-email-screen` with the `Google Sign-In` button and its `Or Layout` and verify that it looks good as well? 🙏

Reply:
👋 @ParaskP7,

I've reviewed fbc7089, LGTM 👍.

Reply:
Thank you @ashiagr ! 🙏

Reply:
> I have update the PR description screenshots to the how the Or Layout is looking now. Can you also please run through the app and verify that everything works as expected, including looking at the login-email-screen with the Google Sign-In button and its Or Layout and verify that it looks good as well? 🙏

@ParaskP7 you may have finally found something that is beyond my eye to spot 🤓  I can't tell, so its totally fine to go with this change if it improves the reusability of this pattern.

Reply:
👋 @osullivanchris !

> @ParaskP7 you may have finally found something that is beyond my eye to spot 🤓 I can't tell, so its totally fine to go with this change if it improves the reusability of this pattern.

Nice, thanks so much for double checking this Chris, and for the 👍 ! We are now good to 🚀